proposal/comp110_architecture_proposal.tex

```diff
@@ -14,61 +14,74 @@
+
+\item[Web link:] \url{http://link.springer.com/chapter/10.1007%2F978-3-662-45523-4_27}
+\item[Full text link:] \url{http://julian.togelius.com/Dahlskog2014Procedural.pdf}
+\item[Comments:] I found this article when looking for another article's pdf file that had a similar name, and I believe this article is relevant because it presents its own unique way to procedurally generate a 2D platformer that I will 
 \end{description}
 
 \section*{Paper 5}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] Procedural Content Generation for Games: A Survey
+\item[Citation:] \cite{hendrikx2013}
+\item[Abstract:] Hundreds of millions of people play computer games every day. For them, game content—from 3D objects to abstract puzzles—plays a major entertainment role. Manual labor has so far ensured that the quality and quantity of game content matched the demands of the playing community, but is facing new scalability challenges due to the exponential growth over the last decade of both the gamer population and the production costs. Procedural Content Generation for Games (PCG-G) may address these challenges by automating, or aiding in, game content generation. PCG-G is difficult, since the generator has to create the content, satisfy constraints imposed by the artist, and return interesting instances for gamers. Despite a large body of research focusing on PCG-G, particularly over the past decade, ours is the first comprehensive survey of the field of PCG-G. We first introduce a comprehensive, six-layered taxonomy of game content: bits, space, systems, scenarios, design, and derived. Second, we survey the methods used across the whole field of PCG-G from a large research body. Third, we map PCG-G methods to game content layers; it turns out that many of the methods used to generate game content from one layer can be used to generate content from another. We also survey the use of methods in practice, that is, in commercial or prototype games. Fourth and last, we discuss several directions for future research in PCG-G, which we believe deserve close attention in the near future.
+
+\item[Web link:] \url{http://dl.acm.org/citation.cfm?id=2422957}
+\item[Full text link:] \url{http://www.st.ewi.tudelft.nl/~iosup/pcg-g-survey11tomccap_rev_sub.pdf}
+\item[Comments:] I found this from the ACM website, but i was unsure whether it would be an appropriate article because it is not specifically about a 2D platformer, but a more general article about Procedural Content Generation.
```

Comment:
Indeed -- however it should be a useful source of background and supporting material for your essay.